tests/phpunit/tests/menu/wp-nav-menu.php

```diff
@@ -0,0 +1,151 @@
+<?php
+
+/**
+ * @group menu
+ */
+class Test_Wp_Nav_Menu extends WP_UnitTestCase {
+
+	protected $menu_id        = 0;
+	protected $lvl0_menu_item = 0;
+	protected $lvl1_menu_item = 0;
+	protected $lvl2_menu_item = 0;
+
+	public function set_up() {
```

Comment:
Does this need to be done before each test, or can it be done before the class (`set_up_before_class()`) so that it only has to be done once?

Reply:
@costdev I didn't thought that `set_up`/`tear_down` would be called for each tests. In this case since we don't mutate the menu in the tests, we could move  the setup code to the `set_up_before_class` method.